packages/manager/modules/sign-up/src/form/details/details.controller.js

```diff
@@ -151,14 +184,7 @@ export default class SignUpDetailsCtrl {
       // When phone country change, the pattern change too.
       // But... the validation is not done automatically.
       // So... be sure that the phone validation is done.
-      const phoneModel = this.formCtrl.phone;
-      const modelValue = phoneModel.$modelValue;
-      const viewValue = phoneModel.$viewValue;
-      const phoneModelValid = phoneModel.$validators.pattern(
-        modelValue,
-        viewValue,
-      );
-      phoneModel.$setValidity('pattern', phoneModelValid);
+      this.constructor.setInputValidity('phone', this.formCtrl);
```

Comment:
Is the comment just above still valid ?

Reply:
Yes it is still valid, as it still sets validity of phone based on the pattern